cwms-data-api/src/test/java/cwms/cda/data/dao/TimeSeriesTextDaoTestIT.java

```diff
@@ -0,0 +1,22 @@
+package cwms.cda.data.dao;
+
+import cwms.cda.api.DataApiTestIT;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Tag;
+
+@Tag("integration")
+class TimeSeriesTextDaoTestIT extends DataApiTestIT {
```

Comment:
I feel like this class name should indicate that it's the base of other test. But saying that it's true of DataApiTestIT itself.

Reply:
I don't think this is the base for anything, but also wasn't added in this PR

Reply:
Yep.